website/versioned_docs/version-18.4.0/README.md

```diff
@@ -0,0 +1,36 @@
+---
+id: cumulus-docs-readme
```

Comment:
Commenting on the README because I can't comment on a directory: this directory should be named version-v18.4.0